components/SearchBar.tsx

```diff
@@ -1,12 +1,14 @@
 "use client";
 
+import { useState } from "react";
 import Icon from "./Icon";
 
-type Props = {
-  onChange: (value: string) => void;
-};
-
-export default function SearchBar({ onChange }: Props) {
+export default function SearchBar({
+  onClick,
+}: {
+  onClick: (value: string) => void;
```

Comment:
```suggestion
  onSearch: (value: string) => void;
```

I think this would make things more clear, since it's only a portion of the component the onClick effects